src/components/ProductsFeatured/index.tsx

```diff
@@ -0,0 +1,62 @@
+import './scss/index.scss';
+
+import { get } from 'lodash';
+import * as React from 'react';
+import { Query } from 'react-apollo';
+import { Link } from 'react-router-dom';
+
+import { Carousel, Loader, ProductListItem } from '..';
+import { generateProductUrl } from '../../core/utils';
+import { GET_FEATURED_PRODUCTS } from './queries';
+
+
+interface ProductsFeaturedProps {
+  title?: string;
+}
+
+const ProductsFeatured: React.SFC<ProductsFeaturedProps> = ({ title }) => {
+  return (
+    <Query
+      query={GET_FEATURED_PRODUCTS}
+      fetchPolicy="cache-first"
+      errorPolicy="all"
```

Comment:
Why do we set `fetchPolicy` and `errorPolicy` props here?

Reply:
@dominik-zeglen In such case it would be nice to suggest which place is more suitable to set those 😉

Reply:
To be honest I don't think we should be setting them at all

Reply:
Why? What approach do we use in Dashboard 2.0?